x-pack/plugins/data_visualizer/public/application/index_data_visualizer/components/index_data_visualizer_view/index_data_visualizer_esql.tsx

```diff
@@ -256,6 +256,7 @@ export const IndexDataVisualizerESQL: FC<IndexDataVisualizerESQLProps> = (dataVi
           detectTimestamp={true}
           hideMinimizeButton={true}
           hideRunQueryText={false}
+          hideQueryHistory
```

Comment:
ℹ️ I am hiding from the data visualizer for now. It works the isLoading logic and to set the errors to the editor to work properly.

Reply:
cc @qn895